### PR TITLE
Aligns description header with screenshot row

### PR DESCRIPTION
**Pull request by JoeButler7**
### What does this PR do?
Fixes the spacing on the `Description` component so that the `Heading` aligns with the `ScreenshotRow`

Requesting review from @jcfilben

### CODE REVIEW

File: src/components/common/Description.js [hunk 1]
```diff
@@ -10,20 +10,30 @@ const Description = ({ title, content, color, images, labels }) => (
         horizontal: 'xlarge',
         bottom: 'medium',
       }}
+      align=""
```
Comment (ShimiSun) [inline src/components/common/Description.js:13]: what's this?

Comment (JoeButler7) [reply src/components/common/Description.js]: Unused code I must have forgotten to remove. It is removed now

Comment (ShimiSun) [reply src/components/common/Description.js]: @JoeButler7 A good practice, you can benefit from in the long run, will be to have your own review done before assigning reviewers, I'm always surprised by the little tweaks I missed once reviewing my own code.

**Comment by JoeButler7** [comment]
### Update:
Made description `Heading` and `Paragraph` responsive so that they align with `ScreenshotRow` and also so that the `ScreenshotRow` will line up with the `ScreenshotRow` in the capabilities section

**Pull request merged**